torchao/testing/model_architectures.py

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
I would clarify these are testing what is effectively a test utility architecture.